src/components/Support/components/ViewSupport/types.ts

```diff
@@ -0,0 +1,9 @@
+import { AnchorHTMLAttributes, ElementType } from "react";
+import { BaseComponentProps } from "../../../types";
+
+export interface ViewSupportProps
+  extends AnchorHTMLAttributes<HTMLAnchorElement>,
+    BaseComponentProps {
+  Icon?: ElementType;
+  url: string;
```

Comment:
Is it possible to use the `href` value on the `AnchorHTMLAttributes` interface instead of adding this `url` value?

Reply:
Yes!

Reply:
If we remove url as a prop, it would be a breaking change.

Reply:
Hi @frano-m how big a change is it both in the package and in the apps. Breaking change is not a problem, but we don't want to introduce a bunch of churn/other difficulties for this. Ahe the "breaking changes" simple/trivial or no?